src/unix/web/matoya.js

```diff
@@ -749,6 +752,7 @@ async function mty_ws_connect(url) {
 		ws.onmessage = (ev) => {
 			ws.msgs.push(ev.data);
 			Atomics.notify(ws.sync, 0, 1);
+			tm = setInterval(() => {ws.send('__ping__');}, 60000); // Fake keepalive
```

Comment:
Q: How do you know every minute should be enough? Presumably you've looked at AWS/CDN settings or w/e and figured out that it drops the connection after X seconds of inactivity, and so you've chosen 60 seconds as a safe number. If this is true, then it would be helpful to add a comment about this.

If not, then you should figure out what the actual timeout threshold is and just the number accordingly, plus add the comment.

Reply:
That information there is worth putting into the code. It will help inform future readers if they ever have to care about this again. I'm send a "suggested change".